po/fa.po

```diff
@@ -0,0 +1,352 @@
+
+#: data/com.mattjakeman.ExtensionManager.appdata.xml.in.in:49
+msgid "Display release notes on first run"
+msgstr "نشان دادن یادداشت های  انتشار در اولین اجرا"
+
+#: data/com.mattjakeman.ExtensionManager.appdata.xml.in.in:50
+msgid "New app icon"
+msgstr "نقشک جدید برنامه"
+
+#: data/com.mattjakeman.ExtensionManager.appdata.xml.in.in:51
+msgid "Optionally sort extensions by enabled status"
+msgstr "مرتب سازی افزونه ها بر اساس فعال بودن (اختیاری)"
+
+#: data/com.mattjakeman.ExtensionManager.appdata.xml.in.in:52
+#: data/com.mattjakeman.ExtensionManager.appdata.xml.in.in:59
+#: data/com.mattjakeman.ExtensionManager.appdata.xml.in.in:64
+msgid "Translation updates"
+msgstr "بروزرسانی های ترجمه"
+
+#: data/com.mattjakeman.ExtensionManager.appdata.xml.in.in:53
+#: data/com.mattjakeman.ExtensionManager.appdata.xml.in.in:84
+msgid "Performance and stability"
+msgstr "عملکرد و پایداری"
+
+#: data/com.mattjakeman.ExtensionManager.appdata.xml.in.in:65
+msgid "Fix special characters in search results"
+msgstr "تصحیح نویسه‌های خاص در نتایج جست‌و‌جو"
+
+#: data/com.mattjakeman.ExtensionManager.appdata.xml.in.in:70
+msgid "Fixes a crash that sometimes occurs while uninstalling an extension"
+msgstr "برطرف کردن خرابی های که بعد از حذف برخی افزونه ها رخ می دهد"
+
+#: data/com.mattjakeman.ExtensionManager.appdata.xml.in.in:75
+msgid "The first feature update to extension-manager. Highlights include:"
+msgstr "اولین به روز رسانی ویژگی به مدیر افزونه ها. نکات برجسته عبارتند از:"
+
+#: data/com.mattjakeman.ExtensionManager.appdata.xml.in.in:77
+msgid "Dark theme and support for overriding the system colour scheme"
+msgstr "تم تیره و پشتیبانی از نادیده گرفتن طرح رنگ سامانه"
+
+#: data/com.mattjakeman.ExtensionManager.appdata.xml.in.in:78
+msgid "Shows screenshots in extension listings"
+msgstr "نشان دادن عکس‌های صفحه در صفحه افزونه ها"
+
+#: data/com.mattjakeman.ExtensionManager.appdata.xml.in.in:79
+msgid "Shell version compatibility check"
+msgstr "بررسی سارگار بودن نسخه گنوم شل"
+
+#: data/com.mattjakeman.ExtensionManager.appdata.xml.in.in:80
+msgid "Localised into seven languages"
+msgstr "ترحمه شده به 7 زبان"
+
+#: data/com.mattjakeman.ExtensionManager.appdata.xml.in.in:81
+msgid "Sort search results"
+msgstr "مرتب کردن نتایج جست‌و‌جو"
+
+#: data/com.mattjakeman.ExtensionManager.appdata.xml.in.in:82
+msgid "Global extension on/off toggle"
+msgstr "خاموش یا روشن کردن سراسری افزونه"
+
+#: data/com.mattjakeman.ExtensionManager.appdata.xml.in.in:83
+msgid "Displays user and system extensions separately"
+msgstr "نشان دادن جدای افزونه های کاربر و سیستمی"
+
+#: src/exm-application.c:106
+msgid "Extension Manager (Development)"
+msgstr "مدیر افزونه‌ها (در حال توسعه)"
+
+#: src/exm-application.c:107
+msgid "Extension Manager"
+msgstr "مدیر افزونه‌ها"
+
+#. TRANSLATORS: 'Name <email@domain.com>' or 'Name https://website.example'
+#: src/exm-application.c:117
+msgid "translator-credits"
+msgstr "نیما قنی اصل <nma.fanniasl@gmail.com> <smartnima.com>"
```

Comment:
Translator credits can either have an email or a website but not both. I would prefer an email here as they don't tend to "expire" like website addresses can.

Also needs a newline since there are two credited translators. Something like this:

```suggestion
msgstr "نیما قنی اصل <nma.fanniasl@gmail.com>\n"
```

Reply:
Hello. I checked.
  I think it would be better to check the "Email or Website Address" section by @nimafanniasl . I myself agree that only the email address should be written.